fed/mosaic-mapping/src/main/java/org/eclipse/mosaic/fed/mapping/ambassador/SpawningFramework.java

```diff
@@ -412,6 +413,8 @@ private void completeSpawnerDefinitions() {
     VehicleTypesInitialization generateVehicleTypesInitialization() {
         HashMap<String, VehicleType> types = new HashMap<>();
 
+        // TODO maybe find a better solution, this is just a first test to see if anything breaks
+        prototypeConfigurations.forEach(p -> types.put(p.name, new VehicleTypeSpawner(p).convertType()));
```

Comment:
@FunKuchen Main problem now comes with predefined SUMO scenarios which cause the failing tests. When we use predefined SUMO scenarios which bring their own vehicle type definitions, we explicitly forbid that a vehicle spawner in mapping_config.json uses a vehicle type which is already defined in the loaded SUMO scenario. The main problem is, that we cannot know from SumoAmbassador perspective which vehicle types are configured in these predefined scenarios. Therefore even merging them somehow would be very difficult, because you'd need to parse all loaded route files and also additional files for vehicle type definitions. So we may need to find a better solution I guess. However, my goal was to see if anything breaks with this line, and luckily we found this already with our integration tests!

A quick (and not so clean) solution would be to add a new property in the MappingConfiguration class called `propagateAllTypes` which we set to `false` in the default case, and only do this line of code when this new property is set to  `true`. As an alternative we could completely close this PR and see if we could fix the problem of missing prototypes in the PHABMACS federate somehow differently. As a third option we could go the full route of defining parking vehicles in the mapping configuration. However, this would require some additional discussions. Therefore we may go the proposed "quick'n'dirty' path and try to find a clean solution in follow up work.